test/extended/images/oc_tag.go

```diff
@@ -0,0 +1,115 @@
+package images
+
+import (
+	"fmt"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Feature:Image] oc tag", func() {
```

Comment:
Do we not already test something like this via the cmd tests?  I thought we did.

Reply:
We did, but some of them are disabled because we don't have the registry for cmd tests in 4.x:

https://github.com/openshift/origin/blob/496eb7cd1b389529b55d2c154421b3be21e93db2/test/extended/testdata/cmd/test/cmd/images.sh#L40

Eventually we are going to get rid of cmd tests (https://jira.coreos.com/browse/DEVEXP-238).